src/pages-and-resources/discussions/data/redux.test.js

```diff
@@ -427,7 +431,7 @@ describe('Data layer integration tests', () => {
           blackoutDates: [],
           // TODO: Note!  As of this writing, all the data below this line is NOT returned in the API
           // but we technically send it to the thunk, so here it is.
-          divideByCohorts: true,
+          divideByCohorts: false,
```

Comment:
why did this change?

Reply:
it has no impact before but now we have added the division schemes so the previous values results in test case failure
so had to change it.